tests/integration/modifier-managers/oo-modifiers-classic-test.js

```diff
@@ -13,11 +13,55 @@ module('Integration | Modifier Manager | oo modifier (classic)', function(hooks)
       this.owner.register(`modifier:${name}`, modifier);
     };
     this.registerModifierClass = (name, ModifierClass) => {
-      this.registerModifier(name, Modifier.modifier(ModifierClass));
+      this.registerModifier(name, ModifierClass);
     };
   });
 
-  module('didInsertElement', function() {
+  module('didInsertElement with calling deprecated Modifier.modifier', function() {
+    test('it has DOM element on this.element', async function(assert) {
+      this.registerModifierClass(
+        'songbird',
+        Modifier.modifier(
```

Comment:
Are we supposed to still use that method here?

Reply:
There's duplication in the tests now.  We call those with Modifier.modifier too, to ensure nothing breaks.